assignment-09-challenge.Rmd

```diff
@@ -190,3 +190,56 @@ ggplot(data = simp_df, aes(x = reorder(site, simpson), y = simpson)) +
     theme_classic()
 ```
 
+2. In assignment 3, we explored a dataset containing changes in yearly plant
+biomass in Abisko National Park, Sweden. In this question, we will use `ggplot2` to
+reproduce a figure in the original paper (Olofsson et al, 2013; [link](https://royalsocietypublishing.org/doi/full/10.1098/rstb.2012.0486)). 
+
+You should still have the dataset from when you completed assignment 3, but
+if not, run the code chunk below to download it.
+
+```{r}
+plant_biomass_url <- 'https://raw.githubusercontent.com/UofTCoders/rcourse/master/data/plant-biomass-preprocess.csv'
+download.file(plant_biomass_url, 'plant-biomass-preprocess.csv')
+plant_biomass <- read_csv('plant-biomass-preprocess.csv')
+```
+
+Reproduce figure 4 from the paper using `ggplot2`. Pay close attention to the
+overall structure of the figure, the scale on the axes, and so on. The colors of
+the points do not need to be the exact same colors as those in the figure, but
+they should be sufficiently close. _(Note: you can ignore the SEM points, and
+the 'look' of your axes does not have to match the figure exactly. The species
+names also not have to be included in the body of the plot as long as they are
```

Comment:
s/also not have/also do not have/

Reply:
Actually - could this final sentence be changed to 'The species names also not have to be included in the body of the plot (like in the paper) as long as they are visible in some form'